### DISCUSSION TITLE
Swift specific Neovim settings

### DISCUSSION THREAD

**Discussion by wojciech-kulik** [wojciech-kulik/xcodebuild.nvim#37] [Tips & Tricks]
If you want to set some specific settings for Swift files like indentation or editor appearance, you can just create a `ftplugin/swift.lua` file in your config directory and put those settings there.

**Comment by guitaripod** [comment]
Do you have an example file with all the possible configs?

**Reply by wojciech-kulik** [reply]
? In this file you can put whatever you like. Example: `vim.opt.tabstop = 4`

**Reply by guitaripod** [reply]
I thought it was some swift specific thing. Like a file that only swift reads from but that doesn't affect something like Go or Python

**Reply by wojciech-kulik** [reply]
This file is to set up Neovim buffers with a specific extension (in this case `swift`).
